### PR TITLE
homepage: infinite looping logo carousel

### PR DESCRIPTION
**Pull request by MananTank**

### CODE REVIEW

**Review by nachoiacovino — COMMENTED** [review]
sorry for the late review :D

File: components/partners/partner-logo.module.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.marqueeGroup {
```
[the remark this answers is not in the document]
Comment (MananTank) [reply components/partners/partner-logo.module.css:1]: I can move the gap variables, but moving the animation would make it more complicated - so better to keep it as pure CSS

File: components/partners/carousel.tsx [hunk 2]
```diff
@@ -1,54 +1,102 @@
...
+
+const gap = { base: "40px", lg: "60px" };
+
+const MarqueeGroup: React.FC<{
+  ariaHidden: boolean;
+  animationDirection?: "normal" | "reverse";
+  children: React.ReactNode;
+}> = ({ ariaHidden, children, animationDirection = "normal" }) => {
+  return (
+    <Box
```
[the remark this answers is not in the document]
Comment (MananTank) [reply components/partners/carousel.tsx:14]: yeah, forgot to change this. Will make it Flex

File: components/partners/carousel.tsx [hunk 3]
```diff
@@ -1,54 +1,102 @@
...
+      maxW="1200px"
+      mx="auto"
+      sx={{
+        "mask-image": `linear-gradient(to right, hsl(0 0% 0% / 0), hsl(0 0% 0% / 1) 10%, hsl(0 0% 0% / 1) 90%, hsl(0 0% 0% / 0));`,
+      }}
+      pt={10}
+      mb={{ base: 20, md: 24 }}
+    >
+      {/* desktop - 1 row with all logos */}
+      <Marquee display={{ base: "none", lg: "flex" }}>
```
[the remark this answers is not in the document]
Comment (MananTank) [reply components/partners/carousel.tsx:64]: yeah, sounds good

**Pull request merged**